### DISCUSSION TITLE
fds multi-fire setup

### DISCUSSION THREAD

**Discussion by ShiChenglong-0423** [firemodels/fds#12764] [Q&A]
I'm currently trying to simulate a fire in a large scene using fds, but I'm having a problem conceptualizing how I can set up two fires in fds with different materials burning in the same scene. For example, I need a piece of grass and a piece of woodland to burn in the same scene, because there is a difference in material between grass and woodland. I have learned so far that running after designing for REAC parameters can only show the burning of the same material, so I would like to ask for some advice. Thank you!

**Comment by rmcdermo** [comment]
You can have multiple fuels.  See the FDS User Guide section in Combustion >> Complex Stoichiometry >> Multiple Fuels.

**Reply by ShiChenglong-0423** [reply]
Thank you for your answer, I read the user guide section but I'm not sure I understand it correctly, I defined one burning material as acetone and the other as methane, now the red part is methane burning and the blue part is acetone burning? 
![微信图片_20240409201222](https://github.com/firemodels/fds/assets/65641602/26940d07-c95c-407c-b7bb-65a8b668dc91)
![微信图片_20240409201320](https://github.com/firemodels/fds/assets/65641602/43c3cf66-b6a1-40cd-8023-d3401baecb85)
Since I'm a beginner, I'm not particularly familiar with fds yet, thanks!

**Reply by rmcdermo** [reply]
Yep, you got it.
